### PR TITLE
Get involved

### PR DESCRIPTION
**Pull request by anibalb2500**
Added the following:
- A button on the home page that goes to the Get Involved page.
- The Get involved page. 
- The "How to apply" Page

The Thinq.tv button on the top of both new pages acts as a home button.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (taangle) [inline .idea/misc.xml]: We don't want to make changes like this to the master branch, I recommend getting rid of this.

File: app/build.gradle [hunk 2]
```diff
@@ -30,5 +30,5 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
 
-    implementation ('org.jitsi.react:jitsi-meet-sdk:2.+') {transitive = true}
+    implementation('org.jitsi.react:jitsi-meet-sdk:2.+') { transitive = true }
```
Comment (taangle) [inline app/build.gradle]: Similarly here, these changes should stay local.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    public void applyNow(View V){
+        Intent i = new Intent(this, getInvolved2.class);
+        startActivity(i);
+    }
+
+
+
+
```
Comment (taangle) [inline app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java]: Get rid of these unnecessary spaces.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+package com.thinqtv.thinqtv_android;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+public class getInvolve extends AppCompatActivity {
```
Comment (taangle) [inline app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java]: Class names should be capitalized.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolved2.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package com.thinqtv.thinqtv_android;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+public class getInvolved2 extends AppCompatActivity {
```
Comment (taangle) [inline app/src/main/java/com/thinqtv/thinqtv_android/getInvolved2.java]: Capitalize this class name, and also rename it to something more descriptive, like GetInvolvedApplyNowPage.

**Review by taangle — CHANGES_REQUESTED** [review]
When renaming things like I requested, make sure you do it with the 'Refactor' option so that the name gets updated everywhere.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolved2.java (hunk 5 above)
Comment (anibalb2500) [reply app/src/main/java/com/thinqtv/thinqtv_android/getInvolved2.java]: Resolved.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java (hunk 4 above)
Comment (anibalb2500) [reply app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java]: Resolved.

File: app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java (hunk 3 above)
Comment (anibalb2500) [reply app/src/main/java/com/thinqtv/thinqtv_android/getInvolve.java]: Resolved.

**Review by taangle — APPROVED** [review]

**Pull request merged**